src/mixins/canvas_events.mixin.js

```diff
@@ -715,12 +725,13 @@
      * @private
      */
     _fireOverOutEvents: function(target, e) {
+      var eventTypePrefix = this.enablePointerEvents ? 'pointer' : 'mouse';
       this.fireSyntheticInOutEvents(target, e, {
         targetName: '_hoveredTarget',
         canvasEvtOut: 'mouse:out',
-        evtOut: 'mouseout',
+        evtOut: eventTypePrefix + 'out',
         canvasEvtIn: 'mouse:over',
-        evtIn: 'mouseover',
+        evtIn: eventTypePrefix + 'over',
```

Comment:
those can stay mouse i guess. Renaming this is gonna break application badly. Those are fabricJS events that devs uses, there is no reason to rename them

Reply:
makes sense -- let's keep the external API as same as possible. I found several other occurrences of the same issue and addressed that in 6c6f4de36597acf786b1ff303112252fc04e5210.